docs/user/welcome/upgrade.rst

```diff
@@ -29,6 +29,21 @@ The first step to upgrade: change the ``geotools.version`` of your dependencies
         ....
     </dependencies>
 
+.. _update32:
+
+GeoTools 32.x
+-------------
+
+NetCDF Version upgrade to 5.5.3
```

Comment:
The documentation should indicate which types of changes and what to do as a GeoTools user:
- Variable names
- Time interpretation
- The need to wipe the index files in the hidden directory and eventually the mosaic configurations and so on.

Could be copied over from the GeoServer docs and adapted (or linked, but maybe the GeoServer docs are interleaving information that's GS specific with the file management bits).

Reply:
Done